spec/system/user/user_profile_reader_stats_spec.rb

```diff
@@ -9,25 +9,25 @@
       end
 
       it "shows the number of visits" do
-        within("div#overall-stats") do
+        within(".sidebar-data") do
           expect(first("#number-visits").text).not_to be_empty
         end
       end
 
       it "shows the number of articles read" do
-        within("div#overall-stats") do
+        within(".sidebar-data") do
           expect(first("#number-articles-read").text).not_to be_empty
         end
       end
 
-      it "shows the number of comments received" do
-        within("div#overall-stats") do
-          expect(first("#number-comments-received").text).not_to be_empty
-        end
-      end
+      # it "shows the number of comments received" do
+      #   within(".sidebar-data") do
+      #     expect(first("#number-comments-received").text).not_to be_empty
+      #   end
+      # end
```

Comment:
this will be uncommented once the method to determine the number of comments made is ready to be added to the view